src/hooks/useCivicProfile.js

```diff
@@ -10,18 +10,83 @@ import { RDF_PREDICATES } from '@constants';
 import useSession from './useSession';
 import useRdfCollection from './useRdfCollection';
 
+const prefix = 'urn:hud:hmis:owl#';
+
+const civicProfileConfig = {
+  firstName: {
+    type: 'string',
+    predicate: 'urn:hud:hmis:owl#FirstName'
```

Comment:
Can't `prefix` be added as a template literal for these four as well?

Reply:
Sure